lucene/analysis/common/src/java/org/apache/lucene/analysis/miscellaneous/ConcatenateGraphFilterFactory.java

```diff
@@ -0,0 +1,61 @@
+/*
+ This software was produced for the U. S. Government
+ under Contract No. W15P7T-11-C-F600, and is
+ subject to the Rights in Noncommercial Computer Software
+ and Noncommercial Computer Software Documentation
+ Clause 252.227-7014 (JUN 1995)
+
```

Comment:
Is this really needed ? I understand that you have some constraint for the development of the main feature but this doesn't seem related so maybe use the usual license ?

Reply:
Good catch; no it's not needed.